Cloud Application Manager/Automating Deployments/running-code-outside-an-instance.md

```diff
@@ -0,0 +1,101 @@
+{{{ "title": "Running Code outside an Instance with Task Variables",
+"date": "26-04-2018",
+"author": "",
+"attachments": [],
+"contentIsHTML": false
+}}}
+
+The event scripts are run, by definition, on the instance they belong to. You may have a need, though, to run some code outside the instance's operating system, either because the code must run before the instance has been provisioned or because it has already been terminated. In these cases you may use **task variables**.
+
+This features is available for boxes that have been [published in the catalog](../Tutorials/publish-script-box.md). 
```

Comment:
Please, check that the link is correctly pointing to the corresponding article once published

Reply:
@gsanchezu We have a build script that checks links automatically